app/src/Command/LDAPImportCommand.php

```diff
@@ -141,15 +147,19 @@ private function importUsers(): void
         $this->em->flush();
     }
 
-    private function createAlias(User $user, string $email): void
+    private function createAlias(User $user, string $aliasEmail): void
     {
-        $alias = $this->em->getRepository(User::class)->findOneBy(['email' => $email]);
+        if ($aliasEmail === $user->getEmail()) {
```

Comment:
Ici par exemple, je pense que ça peut paraître étonnant de voir cette condition, donc peut-être qu’on peut ajouter un commentaire du genre :

```suggestion
        // Make sure to not mark the base user as an alias of himself
        if ($aliasEmail === $user->getEmail()) {
```